ui/ts/pages/nodes.ts

```diff
@@ -309,8 +311,11 @@ module AdminViews {
           primaryContent = ctrl.RenderPrimaryStats();
         }
 
+        let nodeStatus: NodeStatus = nodeStatuses.GetStatus(ctrl.GetNodeId());
+        let updated: number = (nodeStatus ? nodeStatus.updated_at : 0);
```

Comment:
why not use `-Inf` here or throw a `0` into the `.max` before to avoid having both cases?